www/include/Administration/performance/viewData.php

```diff
@@ -90,34 +89,27 @@
 
 $inputs = array();
 foreach ($inputArguments as $argumentName => $argumentValue) {
-    if (!is_null($inputGet[$argumentName]) && trim($inputGet[$argumentName]) != '') {
-        $inputs[$argumentName] = $inputGet[$argumentName];
-    } else {
+    if (!is_null($inputPost[$argumentName]) && trim($inputPost[$argumentName]) != '') {
         $inputs[$argumentName] = $inputPost[$argumentName];
+    } else {
+        $inputs[$argumentName] = $inputGet[$argumentName];
     }
 }
 
-
-if (isset($inputs["searchH"])) {
-    $searchH = $inputs["searchH"];
-    if (isset($oreon->historySearch[$url]) && $oreon->historySearch[$url] != $searchH) {
-        $num = 0;
-        $oreon->historySearch[$url] = $searchH;
```

Comment:
why removing this method ?

Reply:
it was reworked, i explained in detail on chat